elasticdl/python/master/servicer.py

```diff
@@ -185,7 +185,10 @@ def _update_model(self):
         # (grad, var) pairs excluding keras Embedding layer and
         # ElasticDL Embedding layer
         for k in self._gradient_sum:
-            self._gradient_sum[k] = self._gradient_sum[k] / self._grad_to_wait
+            if not self._use_async:
+                self._gradient_sum[k] = (
```

Comment:
How about totally skip the `for` loop and use `zip` to generate the `var` and `grad` tuple ?